### ISSUE TITLE
TTT App - Cancel Button ends up saving instead of cancelling selection

### ISSUE THREAD

**Issue by GenieEH** [emperorshammer/issues-incoming-queue#19]
All cancel buttons in colour selection, actually record your colour change.
To replicate try loading a helmet and change a colour then press cancel in the colour selection screen

**Comment by SmTaUcM** [comment]
Yep, very good spot there. This is a by-product of me setting the colour early to prevent the render queue going crazy if the slider used.

I can fix this for the next release by setting the old colour to memory and monitoring for what the colour selector outputs when it closes.
